SoundSwitch/Framework/NotificationManager/Notification/NotificationBanner.cs

```diff
@@ -46,7 +71,7 @@ public void NotifyProfileChanged(Profile.Profile profile, Bitmap icon, uint? pro
                 Image = icon,
                 Title = string.Format(SettingsStrings.profile_notification_text, profile.Name),
                 Text = string.Join("\n", profile.Devices.Select(wrapper => wrapper.DeviceInfo.NameClean).Distinct()),
-                Position = _bannerPositionFactory.Get(Configuration.BannerPosition)
+                Position = BannerPosition()
```

Comment:
Then this get changed to this
```suggestion
                Position = BannerPosition
```